include/caf/net_state.rst

```diff
@@ -0,0 +1,38 @@
+.. _caf_net_state:
```

Comment:
Please move the file to: `doc/nrf/libraries/caf/` to be consistent with other CAF modules.

Reply:
ok